src/translations/en/feedback.json

```diff
@@ -10,13 +10,14 @@
     "delete_identifier": "Could not delete {{qualifier}}",
     "delete_registration": "Could not delete registration",
     "download_file": "Could not download file",
+    "get_affiliation_name": "Could not retrieve affiliation name for ",
```

Comment:
```suggestion
    "get_affiliation_name": "Could not retrieve affiliation name for {{unitUri}}",
```

Reply:
prøvde med det først, men i18n klarer ikke å tolke URLen som link, så det blir masse `&&%=`"""`

Reply:
ah stemmer, hadde akkurat samm problemet selv nylig, kan evt løses med `interpolation: { escapeValue: false }`
https://github.com/BIBSYSDEV/NVA-Frontend/blob/bf33dd934af127c3b7b2d1b85b81b61d9a5db4ee/src/utils/hooks/useFetch.ts#L26

Reply:
ah, good to know! da bruker jeg den der 😊